### DISCUSSION TITLE
Add new plot during farmer running.

### DISCUSSION THREAD

**Discussion by coaxs** [Chia-Network/chia-blockchain#1798] [Q&A] [answered]
Is it possible using cli add new plots during farmer is runing or I need to restart all services? 

chia start farmer -r

**Comment by hoffmang9** [comment, marked as answer]
The plotter automatically adds plots to the running farmer.

**Reply by coaxs** [reply]
Yep you are right when Farming status is "Farming" new plots started to be added automatically.
